src/web/components/ParticipantRequests/ParticipantRequestItem.scss

```diff
@@ -8,6 +8,13 @@
     margin-right: 8px;
   }
 
+  .participant-request-api-role-label {
+    background: var(--theme-label);
```

Comment:
An alternative to the more severe refactoring above would to extract some of the styling within this file, e.g. something like:
```
.participant-request-label {
  border-radius: 3px;
  padding: 4px 8px;
  margin-right: 8px;
  background: var(--theme-label-background);
}

.participant-request-type-label {
  @extend .participant-request-label;
}

.participant-request-api-role-label {
  @extend .participant-request-label;
  background: var(--theme-label);
}
```